packages/core/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@viamrobotics/prime-core",
+  "version": "0.0.1",
+  "scripts": {
```

We'll want a publish config section (there are a few ways to do this, `publishConfig` is my favorite because it's portable) to tell `npm` on the initial publish that this namespaced package should be public, because the npm default for namespaced packages is private

```suggestion
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
```